plugins/build-your-own.rst

```diff
@@ -4,7 +4,7 @@ Building Custom Plugins
 When writing your own Plugin, you need to be aware that the Plugin Client is async first.
 This means that every plugin must be written with Promises. More about this later.
 
-Each plugin must implement the ``Http\Client\Plugin\Plugin`` interface.
+Each plugin must implement the ``Http\Client\Common\Plugin`` interface.
```

Comment:
right, good point. so lets be even more explicit and say something like

The plugins where moved to the clients-common package in version 1.1. If
you work with version 1.0, you need to require the separate plugin
package. The new interface will keep implementing
``Http\Client\Plugin\Plugin`` until version 2 but relying on the old
interface is deprecated.